python/setup.py

```diff
@@ -275,7 +275,7 @@ def get_packages(self):
             else "grpcio",
         ],
         "serve": [
-            "uvicorn",
+            "uvicorn[standard]",
```

Comment:
This will bring in "required" dependency on uvloop

Reply:
I believe `uvicorn[standard]` will fail to install on windows. I'm not sure how to accomplish this, but can we continue specifying vanilla `uvicorn` for windows installations?

Reply:
Seems like our Serve tests running on Windows seem to pass in CI